docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -73,37 +92,34 @@ To stop an individual tunnel via the command line/prompt, you must send some sor
 Windows has no "signals" command the way Linux/Unix/MacOS does, instead they use TaskKill iirc, for example: `taskill /PID 1234`.
```

Comment:
Tested the Windows command, and it works similarly to `SIGKILL`. Could reword to something like this:

Windows does not have signals the way Linux/Unix/MacOS does. Instead, `taskkill` can be used to immediately disrupt the process, similar to sending a`SIGKILL` signal:
 ```bash
taskkill /pid 1234 
```